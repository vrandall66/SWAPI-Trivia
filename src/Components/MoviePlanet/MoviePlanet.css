.planet {
  background-color: blue;
  border-radius: 100%;
  color: pink;
  height: 200px;
  width: 200px;
}