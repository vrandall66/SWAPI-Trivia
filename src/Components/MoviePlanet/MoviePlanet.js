import React from "react";
import "./MoviePlanet.css";

const MoviePlanet = ({ movie, updatePlanetModalState, updateCurrentMovie }) => {
  return (
    <div
      key={movie.id}
      className="planet"
      movieData={movie}
      onClick={() => {
        // updatePlanetModalState()
        updateCurrentMovie(movie)
      }
    }
    >
      <h1>{movie.id}</h1>
    </div>
  );
};

export default MoviePlanet;
