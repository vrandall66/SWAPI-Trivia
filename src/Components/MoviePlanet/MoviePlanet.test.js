import React from "react";
import { shallow } from "enzyme";
import MoviePlanet from "./MoviePlanet";

describe("MoviePlanet", () => {
  let wrapper;
  let movie = {
    id: 0,
    releaseDate: "",
    title: ""
  };
  let index = 0;
  let updatePlanetModalState = jest.fn();
  let updateCurrentMovie = jest.fn();
  let resetMovieState = jest.fn();

  beforeEach(() => {
    wrapper = shallow(
      <MoviePlanet
        movie={movie}
        key={index}
        updatePlanetModalState={updatePlanetModalState}
        updateCurrentMovie={updateCurrentMovie}
        resetMovieState={resetMovieState}
      />
    );
  });

  it("should match the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
