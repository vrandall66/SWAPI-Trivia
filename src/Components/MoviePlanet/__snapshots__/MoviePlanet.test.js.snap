// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePlanet should match the snapshot 1`] = `
<div
  className="planetImage planetundefined"
  key="0"
  onClick={[Function]}
>
  <h1
    className="planetNumber"
  />
</div>
`;
